openbb_terminal/cryptocurrency/due_diligence/coinglass_model.py

```diff
@@ -13,7 +13,8 @@
 
 api_url = "https://open-api.coinglass.com/api/pro/v1/"
 
-INTERVALS = [0, 1, 2, 4]
+intervals = [0, 1, 2, 4]
+INTERVALS = [str(x) for x in intervals]
```

Comment:
Can you add a comment here why you are making this a string so that when we look at this code in a year we arent super confused